playbooks/openstack/inventory.py

```diff
@@ -130,24 +152,14 @@ def build_inventory():
         server for server in cloud.list_servers()
```

Comment:
Could the `cluster_hosts` loop be included in the `base_openshift_inventory()` function as well? Pass in the server list and return an inventory and/or the cluster_hosts list (if necessary)?